api/metrics/taxitrips.py

```diff
@@ -6,9 +6,9 @@ def __init__(self, connection):
         self.connection = connection
         
     #returns avergae trip speed per pickup area    
-    def avg_speed_per_pickup(self):
+    def get_avg_speed_per_pickup(self):
         query = """
-        SELECT pickup_community_area, AVG(trip_miles/(trip_minutes/60)) as avg_speed
+        SELECT pickup_community_area as area_number, AVG(trip_miles/(trip_minutes/60)) as value
```

Comment:
Code style preference: In the future, I would recommend putting each selected expression on its own line to make the query easier to read.